rpm/README.md

```diff
@@ -0,0 +1,43 @@
+# System Admin Toolkit Offline Documentation
+
+The SAT documentation can be found online at the following url:
+https://cray-hpe.github.io/docs-sat
+
+Additionally, it is available offline in markdown format, which may be viewed
+with a markdown viewer or simply with a text editor. A table of contents can be
+found below.
```

Comment:
```suggestion
It is also available offline as markdown, which may be viewed with a markdown viewer or with a text editor. A table of contents can be
found below.
```

Reply:
By "markdown viewer" we mean something that can render markdown, right?

Reply:
Correct. The suggestion looks good to me, fixed with: 22d69ae182d7bcc288d76a4bc0afb2848ee9ea29